src/docker-compose.yml

```diff
@@ -10,3 +10,13 @@ services:
       - POSTGRES_PASSWORD=
     ports:
       - 5432:5432
+  keycloak:
+    image: jboss/keycloak
```

Comment:
Do you need to pin the version here, I see in master.json it defines version 7.0.0

Reply:
👍